docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -282,3 +282,59 @@ DSA keys look almost identical but begin with ``ssh-dss`` rather than
 
     :raises cryptography.exceptions.UnsupportedAlgorithm: If the serialized
         key is of a type that is not supported.
+
+Serialization Formats
+~~~~~~~~~~~~~~~~~~~~~
+
+.. class:: Format
+
+    .. versionadded:: 0.8
+
+    An enumeration for key formats.
+
+    .. attribute:: TraditionalOpenSSL
+
+        Frequently known as PKCS#1 format.
+
+    .. attribute:: PKCS8
```

Comment:
More words are needed here. Should probably specifically mention that where possible PKCS8 is prefered since it allows better encryption algorithms.